Assets/MRTK/Core/MixedRealitySceneContent.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could you update this header to the latest version? Also could you remove the multiple #defines below as we are only supporting Unity 2018.4 and later?